patches/coreboot.git/shebangs.patch

```diff
@@ -0,0 +1,8 @@
+--- a/payloads/external/skiboot/skiboot/make_version.sh	2023-06-09 11:46:42.694892749 -0400
++++ b/payloads/external/skiboot/skiboot/make_version.sh	2023-06-09 12:05:16.420164712 -0400
+@@ -1,4 +1,4 @@
+-#!/bin/bash
++#!/usr/bin/env bash
+ 
+ usage() {
+ 	echo "$0 [ prefix ]"
```

Comment:
This patch won't apply to anything since the coreboot modules are handled differently now, but I think it is superseded by patches/coreboot-talos_2/0001-Use-usr-bin-env-in-skiboot.patch and can be removed.

Reply:
:+1: fixed and pushed